src/sdk-installation/cocos.mdx

```diff
@@ -84,10 +84,59 @@ SDK will upload sessions **only on WiFi**. When uploading on mobile connections
 
 ## iOS
 
-Coming soon
+1. In projects that use **Cocoapods**, Smartlook can be installed by adding the respective pod into `Podfile`:
+
+<Code
+  kind="example"
+  snippets={{
+    ruby: `
+pod Smartlook
+`,
+  }}
+/>
+
+Smartlook can be also added directly to the app project by **downloading** the latest [Smartlook iOS SDK](https://sdk.smartlook.com/ios/smartlook-ios-sdk-1.4.1.2356.zip), unzipping the file and adding `Smartlook.framework` to the Xcode project.
+
+2. To **setup** and **start** recording with Smartlook, Smartlook must be imported and configured e.g., in `AppDelegate` and its `didFinishLaunching`:
+
+<Code
+  kind="example"
+  snippets={{
+      swift: `
+import Smartlook
+...
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {\n
+    Smartlook.setup(key: "YOUR_API_KEY", options: [.useAdaptiveFramerate: false]))
+    Smartlook.startRecording()
+      `,
+      'objective-c': `
+#import <Smartlook/Smartlook.h>
+...
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {\n
+    [Smartlook setupWithKey: @"YOUR_API_KEY" options:@{SLSetupOptionUseAdaptiveFramerateKey: @NO }];
+    [Smartlook startRecording];
+      `,
+    }
+  }
+/>
+
+<TextBlock kind="important">
+{`
+To setup Smartlook, the **unique project API key **is needed. It can be found in the mobile project settings in <Link to="https://app.smartlook.com/settings/projects">Smartlook Dashboard</Link>.\n
+To understand the *adaptive framework* option see <Link to="/docs/sdk/conceptual/screen-recording-quality" platform="cocos">Smartlook API reference</Link>. 
```

Comment:
You can do adaptive recording even on game engine? :O

Reply:
Adaptive framerate is independent on the rendering engine in the standard SDK build which is used here (also because Cocos apps seems to be a mixed UI of UIKit and Metal or OpenGL.).  Adaptive framerate suspends itself in standard SDK when a known non-UI-changes-traceable kind of view is present in the views hierarchy. This does not work here in for at least one customer, presumably because their app renders to OpenGL which we do not detect, as it is obsolete on iOS. If we switch the adaptive framerate off for some frameworks, it is done in the respective bridge, which we do not provide for Cocos.